rosidl_runtime_py/rosidl_runtime_py/convert.py

```diff
@@ -85,41 +92,67 @@ def to_string(val):
                         val += '...'
             r = str(val)
         else:
-            r = message_to_csv(val, truncate_length)
+            r = message_to_csv(val, truncate_length, no_arr, no_str)
         return r
     result = ''
     # We rely on __slots__ retaining the order of the fields in the .msg file.
     for field_name in msg.__slots__:
         value = getattr(msg, field_name)
+        field_type = msg._fields_and_field_types[field_name[1:]]
+
         if result:
             result += ','
-        result += to_string(value)
+
+        if no_arr is True and any(
+                isinstance(value, t) for t in [list, tuple, array.array, numpy.ndarray]):
+            result += '<array type: <{0}>, length: <{1}>>'.format(
+                field_type, len(value))
```

Comment:
@vinnamkim when the type is already bounded, length information turns out to be redundant. Consider dropping it in that case.

Reply:
Fixed